src/ripple/app/main/GRPCServer.h

```diff
@@ -21,6 +21,7 @@
 #define RIPPLE_CORE_GRPCSERVER_H_INCLUDED
 
 #include <ripple/app/main/Application.h>
+#include <ripple/beast/core/CurrentThreadName.h>
```

Comment:
This may be a little counter intuitive, but in my opinion this include file is actually better in the `.cpp` file than in the `.h` file.  There are two reasons for that:

1. Nothing in the `.h` file is dependent on the contents of CurrentThreadName.h.  You should limit include files to only those places where they are needed.

2. Any `#include` in a header file is promiscuous.  Because header files are included lots of places, including CurrentThreadName.h in GRPCServer.h means that any file that includes GRPCServer.h will also `#include` CurrentThreadName.h.  And then there are includes of includes of includes.  You get the picture.

    Contrariwise, people don't (typically) do a `#include` of a cpp file.  So if you put the `#include` in GRPCServer.cpp then no additional files will be infiltrated by CurrentThreadName.h.

I hope that makes sense.  If not, then ask away...